openlibrary/core/models.py

```diff
@@ -20,7 +20,7 @@
 from openlibrary.core.vendors import create_edition_from_amazon_metadata
 
 # Seed might look unused, but removing it causes an error :/
```

Comment:
this comment looks relevant:
`# Seed might look unused, but removing it causes an error :/`